dbms/src/Flash/ResourceControl/LocalAdmissionController.h

```diff
@@ -580,6 +603,7 @@ class LocalAdmissionController final : private boost::noncopyable
         = std::chrono::steady_clock::now();
 
     ::pingcap::kv::Cluster * cluster = nullptr;
+    std::atomic<bool> need_reset_unique_client_id{false};
```

Comment:
When will this be false? Looks like it is always true once `unique_client_id` is initialized?

Reply:
This is to avoid `deleteServerIDFromGAC()` is called before `acquireServerIDFromGAC()` succeed.